src/homekit_decl.c

```diff
@@ -89,12 +89,12 @@ homekit_accessory_t *accessories[] = {
                     NULL
                     }),
             HOMEKIT_SERVICE(LIGHTBULB, .primary=false, .characteristics=(homekit_characteristic_t*[]){
-                    HOMEKIT_CHARACTERISTIC(NAME, "ratgdo"),
```

Comment:
What was the reasoning behind this change?   It seems unrelated to this changeset.  Also, I had originally done it this way and something made me change it, but I honestly don't remember why.  If the change needs to be there, lets break it out into a different commit with a comment describing why we made the change.

Reply:
i changed this to have it make more sense in the homekit app on my iphone
it shows a icon of a light bulb so might as well call it lights, and that's what it is
same for motion
but if needs be we can make this separate PR

Reply:
Hmm... I don't actually see this text anywhere in HomeKit.  Are you suggesting it names the accessories differently based on this value?

Reply:
remove the device from your home kit, and re-add 
it gets shown (at least on my iphone) when i add new device
i dont have appletv or homepod... i just use iphone
so not sure how things work in HK on those devices

Reply:
I've done that numerous times and the Light shows up as "Light" and the motion detector says "Motion Sensor".  I just tried to pair on another device where I don't have a hub and see the same thing.  I'm not opposed to the change, but we should move it to a different commit, and I'm still very curious about it.  I had it this way to begin with.  Not sure my reasoning for renaming all services the same way now, but this way seems more correct.

Reply:
i know i wouldnt have changed it to light and motion, if it already said that :)
it showed a lightbulb and said ratgdo and that just looked wierd
since its of HK type lightbulb it prob should say that :)
maybe i can screen shot what it did
me temp change it back and flash, and try

Reply:
ha, well course i have 1.0 and no motion, but light is saying light and i flashed it with ratgdo in there :)
ok, reverting back to ratgdo... and keeping eye on it :)

Reply:
I tend to side with @jgstroud on this one.

There are actually two settings in HomeKit, which have evolved over the years.  There is `Characteristic.Name` and `Characteristic.ConfiguredName`.  Configured name is often empty/undefined and I think is a fairly recent add (like in the last 5 years).  A value is written to that by Apple Home if a user enters a "name" for the device/accessory.  In absence of ConfiguredName, the apps will use the Name value, but on change will save the change to ConfiguredName.

But what should the default be?  Typically you do not name a light "light" or a switch "switch" as the device type is identified by what it is.  Typically you name it something like "Kitchen Counter" or "Family Room Desk" (and note that in Apple Home, if the start of the name is identical to the room it is in, then that is removed from the displayed name.... so the app will only display "Desk" for the lamp that is named "Family Room Desk".

For newly installed devices normal practice seems to be to name the device something unique so that it can easily be identified when newly installed.  Typically a combination of device manufacture and serial number or last 4 digits of MAC address, or something like that.  It is therefore appropriate for us to set everything to "ratgdo" or better still, a combination of "ratgdo" + MAC address...  similar to what we now have for the web page title.

Reply:
I might go back on this.  For some reason starting today while trying to reproduce the pairing failures I am now seeing the door and light show up as ratgdo and ratgdo2